mysql-test/t/bug_ps7923.test

```diff
@@ -0,0 +1,36 @@
+--source include/count_sessions.inc
+
+--connect (con1,localhost,root,,)
+--connect (con2,localhost,root,,)
+
+--connection default
+
+DELIMITER //;
+
+CREATE PROCEDURE my_infinite_alter_ts()
+BEGIN
+  WHILE TRUE DO
+    ALTER TABLESPACE mysql ENCRYPTION='Y';
+    ALTER TABLESPACE mysql ENCRYPTION='N';
+  END WHILE;
+END//
+
+DELIMITER ;//
+
+--connection con1
+--send CALL my_infinite_alter_ts()
+
+--sleep 5
+
+--connection default
+
+--let $do_not_echo_parameters = 1
+--let $restart_parameters = restart: --gtid-mode=ON --enforce-gtid-consistency=ON --early-plugin-load=keyring_file=$KEYRING_PLUGIN --loose-keyring_file_data=$MYSQL_TMP_DIR/keyring $KEYRING_PLUGIN_OPT
+--source include/kill_and_restart_mysqld.inc
+
+--disconnect con1
+--disconnect con2
```

Comment:
Not needed if con2 is not used

Reply:
Removed (along with corresponding `connect` above).